src/test/FuzzerImpl.cpp

```diff
@@ -97,6 +123,79 @@ makeAssetCode(int digit)
     return code;
 }
 
+void
+generateStoredLedgerKeys(StoredLedgerKeys& keys)
+{
+    size_t constexpr NUM_UNVALIDATED_LEDGER_KEYS = 0x08U;
+    static_assert(NUM_UNVALIDATED_LEDGER_KEYS < NUM_STORED_LEDGER_KEYS,
+                  "No room for valid ledger keys");
+
+    auto const firstUnvalidatedLedgerKey =
+        NUM_STORED_LEDGER_KEYS - NUM_UNVALIDATED_LEDGER_KEYS;
+
+    for (auto index = 0; index < firstUnvalidatedLedgerKey; ++index)
+    {
+        keys[index] =
+            LedgerEntryKey(LedgerTestUtils::generateValidLedgerEntry());
+    }
+
+    for (auto index = firstUnvalidatedLedgerKey; index < NUM_STORED_LEDGER_KEYS;
+         ++index)
+    {
+        size_t const entrySize = 3;
+        keys[index] = autocheck::generator<LedgerKey>()(entrySize);
+    }
+}
+
+void
+storeLedgerKey(StoredLedgerKeys& keys, LedgerKey const& key)
+{
+    auto byte = getShortKey(key);
+    keys[byte % NUM_STORED_LEDGER_KEYS] = key;
+}
+
+void
+storeLedgerKeyIDsForOp(StoredLedgerKeys& keys,
+                       std::shared_ptr<OperationFrame const> opFrame)
+{
+    auto op = opFrame->getOperation();
+    auto const opType = op.body.type();
+
+    // For any operation type that we choose to intercept here, we can add to
+    // the set of stored LedgerKeys any LedgerKey that the operation makes
+    // available for reference by future operations, and give it a short ID by
+    // which future fuzz operations can refer to it (by calling getShortKey()).
+    switch (opType)
```

Comment:
When I started this PR I thought it would take more code than it turns out to appear to to support storing ledger keys for each new operation type.  Now that I look at its current form, it seems that each op type we support is just one more one-line case in that switch statement.  So I'm happy to bring offers and trust lines into the scope.  I've added them.